src/change_stream/mod.rs

```diff
@@ -107,6 +111,28 @@ pub struct ChangeStream {
 }
 
 impl ChangeStream {
+    pub(crate) fn new(
+        mut cursor: Cursor,
```

Comment:
This should just take a `ChangeStreamSpecification` rather than taking the individual fields of it passed in separately

Reply:
didn't push earlier. fixed now